lib/samples/create_buffers_around_points/create_buffers_around_points.dart

```diff
@@ -0,0 +1,446 @@
+import 'dart:math';
```

Comment:
We need to get the standard copyright block at the top of this file. I think we have a git hook that takes care of it, though maybe not for the samples repo.

Reply:
Done Added, the copyright block.